source/reference/api/reverse.txt

```diff
@@ -159,4 +159,6 @@ Limitations
 
 The ``reverse`` endpoint does not support :ref:`filtered sync <c2c-filtered-sync>`.
 
+Mongosync ``reverse`` mode is not compatible with any beta features. 
```

Comment:
The team clarified that Operational Rollover Resilience is compatible with Reverse mode, and the rest of the beta features are not compatible with it. We can add that here - @ajayvijayakumar123 please correct me if I am mistaken. Thank you!

Reply:
Thank you for making the change! I see Ajay is OOO so may not have seen my above message, @edobranov would you be able to confirm that my understanding is correct? Thank you!

Reply:
Yes this looks correct to me!